src/processTargets/marks/LineNumberStage.ts

```diff
@@ -16,13 +17,70 @@ export default class implements MarkStage {
       return [];
     }
     const editor = context.currentEditor;
-    const anchorLine = getLine(editor, this.modifier.anchor);
-    const activeLine = getLine(editor, this.modifier.active);
-    const anchorRange = editor.document.lineAt(anchorLine).range;
-    const activeRange = editor.document.lineAt(activeLine).range;
-    const contentRange = anchorRange.union(activeRange);
-    const isReversed = this.modifier.anchor < this.modifier.active;
-    return [createLineTarget(editor, isReversed, contentRange)];
+    if (this.modifier.anchor && this.modifier.active) {
+      const anchorLine = getLine(editor, this.modifier.anchor);
+      const activeLine = getLine(editor, this.modifier.active);
+      const anchorRange = editor.document.lineAt(anchorLine).range;
+      const activeRange = editor.document.lineAt(activeLine).range;
+      const contentRange = anchorRange.union(activeRange);
+      const isReversed = this.modifier.anchor > this.modifier.active;
+      return [createLineTarget(editor, isReversed, contentRange)];
+    }
+    const targetDescriptor = {
+      type: "list" as const,
+      elements:
+        this.modifier.elements?.map((element) => ({
+          type: "range" as const,
+          anchor: {
+            type: "primitive" as const,
+            mark: {
+              type: "lineNumber" as const,
+              anchor: element.anchor,
+              active: element.anchor,
+            },
+            modifiers: [],
+          },
+          active: {
+            type: "primitive" as const,
+            mark: {
+              type: "lineNumber" as const,
+              anchor: element.active,
+              active: element.active,
+            },
+            modifiers: [],
+          },
+          excludeAnchor: element.excludeAnchor,
+          excludeActive: element.excludeActive,
+          rangeType: element.rangeType,
+        })) ?? [],
+    };
+    return processTargets(context, [targetDescriptor])[0] as LineTarget[];
+
+    // const lineTargets = this.modifier.elements?.map((element) => {
```

Comment:
All of this could be done with `targetsToContinuousTarget` once that is merged.
https://github.com/pokey/cursorless-vscode/blob/d277b5b3b1a16c9e1a886432ddaec3f1ed2628b6/src/processTargets/processTargets.ts#L88-L108